Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/AssetBrowserModel.cpp

```diff
@@ -220,18 +221,29 @@ namespace AzToolsFramework
             {
                 return false;
             }
+
             Path newPath = oldPath;
             newPath.ReplaceFilename(newFile);
             newPath.ReplaceExtension(extension);
             using SCCommandBus = AzToolsFramework::SourceControlCommandBus;
             SCCommandBus::Broadcast(
                 &SCCommandBus::Events::RequestRename, oldPath.c_str(), newPath.c_str(),
-                [this, index](bool success, [[maybe_unused]] const AzToolsFramework::SourceControlFileInfo& info)
+                [&, index, item, newPath](bool success, [[maybe_unused]] const AzToolsFramework::SourceControlFileInfo& info)
                 {
                     if (success)
                     {
                         emit dataChanged(index.parent(), index);
+						
+						if (m_assetEntriesToCreatorBusIds.contains(item))
```

Comment:
nit: spacing looks wrong in the diff, does it look okay in your editor?

Reply:
It looks okay in my editor, but there was empty space on the line above it.

Just removed the space. Will see if this next commit is better.

Reply:
Confirmed removing the empty space on the line above it fixed the diff.